metadata.json

```diff
@@ -36,7 +36,7 @@
   "operatingsystem_support": [
     {
       "operatingsystem": "Debian",
-      "operatingsystemrelease": ["11","12"]
```

Comment:
Why drop 11?

Reply:
Looks like I blindly removed the first array element. Fixed, thanks.